hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/StreamFilesPTest.java

```diff
@@ -79,6 +79,11 @@
 
     @Before
     public void before() throws Exception {
+        // On Windows, "file changed" events are not delivered unless enough data is written
+        // or the file is closed. Some tests in this class rely that after a flush, the line is picked
+        // up by the processor.
+        assumeThatNoWindowsOS();
```

Comment:
You could do this check in `@BeforeClass` and avoid the nullcheck in `after()`

Reply:
The nullcheck in `after()` is good anyway if the creating of workdir or of the thread fails... I'm lazy now, I'm merging :)